tests/shared/test_progress_notifications.py

```diff
@@ -41,7 +41,7 @@ async def run_server():
                 capabilities=server.get_capabilities(NotificationOptions(), {}),
             ),
         ) as server_session:
-            global serv_sesh
+            global serv_sesh  # noqa: PLW0603
```

Comment:
As above, would be great not to be adding #noqa statements. Can we improve the design instead?